lib/submit-analytics.js

```diff
@@ -23,6 +42,74 @@
 
 'use strict';
 
+//const { resolve } = require('path');
+let utils = require('./utils');
+
+const nodeRequire = require;
```

Comment:
If this file is never supposed to be loaded by a React Native app, perhaps this could just be left out and `require` used directly (as is the case in the `fetchPlatformData` function already)?
```suggestion
```